### ISSUE TITLE
Enhancement suggestion: Ability to select labels

### ISSUE THREAD

**Issue by SantoFantasy** [kube-green/kube-green#321]
**Suggestion:** ability to select by label folowing this example:
```yaml
apiVersion: kube-green.com/v1alpha1
kind: SleepInfo
metadata:
  name: working-hours
spec:
  weekdays: "1-5"
  sleepAt: "23:0"
  wakeUpAt: "07:00"
  suspendDeployments: true
  suspendCronJobs: false
  selector:
    matchLabels:
      app: my-app
```

**Reasoning:** When using the current configuration, the behavior is to put the entire cluster to sleep and we select the apps that we don't want to be shut down with the `excludeRef`. With a label selector, it's possible to fine tune more easily wich deployments are put to sleep, enabling easy adoption on large scale clusters and ensuring nothing critical is being turned off.

**Comment by davidebianchi** [comment]
Hi @SantoFantasy! I think it's a good idea, so an allow list of labels to sleep? This configuration should be cluster wide? Probably it could be more useful. Something like the CRD proposed [here](https://github.com/kube-green/kube-green/issues/167).

At the moment, we support the match label selector only in the excludeRef: https://kube-green.dev/docs/apireference_v1alpha1/#kube-green.com/v1alpha1.ExcludeRef

**Comment by SantoFantasy** [comment]
Hi
Yes, an allow/include list, keeping the current behaviour (namespace wide) for the SleepInfo. The #167 CRD proposal would also benefit from an allow list like this one.

Sadly, the excludeRef is not a viable option for my case.

**Comment by Valefoth** [comment]
Hi, I think an `includeRef` would be a good way to specify only some resources for sleep hours.
```yaml
includeRef:
 - matchLabels: 
   kube-green.dev/include: true
```
What do you think of that ?

**Comment by k11h-de** [comment]
I would love to see this feature / enhancement.

**Comment by davidebianchi** [comment]
There is the #445 which add the includeRef option!

**Comment by davidebianchi** [comment]
Closed with #445. It is released in the v0.6.0-rc.2
